### PR TITLE
draco 1.2.5 (new formula)

### PR DESCRIPTION
**Pull request by hobu**
Initial support for Google Draco, a C++ point cloud and mesh compression library. https://github.com/google/draco

Checklist
-----

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by hobu** [comment]
Should I expect this package to make it into homebrew-core, or should I submit it to another tap? If so, can you please suggest where it might be appropriate?

File: Formula/draco.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  url "https://github.com/google/draco/archive/1.2.4.tar.gz"
+  sha256 "993c78880f4bbd2bfbc7b1e16809c2f62f8ecf9040f68db75ca65be3ac34be4f"
+
+  depends_on "cmake" => :build
+
+  def install
+    mkdir "build" do
+      args = std_cmake_args
+      args << "-DENABLE_TESTS=ON"
+      system "cmake", "..", *args
```
Comment (fxcoudert) [inline Formula/draco.rb]: Let's not enable tests please (we only do that for security software). Thus:

`system "cmake", "..", * std_cmake_args`

File: Formula/draco.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+      system "make", "install"
+    end
+  end
+
+  test do
+    shell_output("#{bin}/draco_encoder --help", 255)
```
Comment (fxcoudert) [inline Formula/draco.rb]: We will need a test that actually exercises the software's functionality.

File: Formula/draco.rb [hunk 3]
```diff
@@ -0,0 +1,21 @@
+class Draco < Formula
+  desc "3D geometric mesh and point cloud compression library"
+  homepage "https://google.github.io/draco/"
+  url "https://github.com/google/draco/archive/1.2.4.tar.gz"
```
Comment (fxcoudert) [inline Formula/draco.rb]: Sorry it took long to review… please update to latest version, 1.2.5

**Review by fxcoudert — CHANGES_REQUESTED** [review]

**Comment by fxcoudert** [comment]
We will need a test that actually exercises the software's functionality.

**Comment by hobu** [comment]
> We will need a test that actually exercises the software's functionality.

I wasn't sure what to do here. It's just a support library.

**Comment by fxcoudert** [comment]
Does it include a source example or test file that we could install (in `pkgshare`), then compile and run as a test? Otherwise, we can include a very simple C file (see for example what we do in `gmp`).

**Comment by fxcoudert** [comment]
Thanks @hobu for the pull request!

**Review by fxcoudert — APPROVED** [review]

**Pull request closed without merging**